src/app/modules/main/wallet_section/all_tokens/view.nim

```diff
@@ -128,3 +128,28 @@ QtObject:
   proc currencyFormatsUpdated*(self: View) =
     self.flatTokensModel.currencyFormatsUpdated()
     self.tokensBySymbolModel.currencyFormatsUpdated()
+
+  proc tokenPreferencesUpdated*(self: View, result: bool) {.signal.}
```

Comment:
@Cuteivist I don't see any interaction with the qml for this and all below added signals/slots?